workers/jobs/scraping.go

```diff
@@ -0,0 +1,20 @@
+package jobs
```

Comment:
Avoid `-ing` in the naming job, prefer the imperative tone `scrap_keyword_job.go` or using a noun `keyword_scraper.go`

Reply:
Fixed in 60390a66288acf8ac626261273fbf39b8aca1115